shared/src/packages/users/libs/validation-schemas/user-sign-in.validation-schema.ts

```diff
@@ -0,0 +1,24 @@
+import joi from 'joi';
+
+import { UserValidationMessage } from '../enums/enums.js';
+import { type UserSignInRequestDto } from '../types/types.js';
+
+const userSignIn = joi.object<UserSignInRequestDto, true>({
```

Comment:
Please contact with developers of singn in and sign up, bcos we decided to create a common validation schema

Reply:
Yelyzaveta said to make separate ones)